pkg/resources/view_acceptance_test.go

```diff
@@ -770,6 +770,118 @@ end;;
 	})
 }
 
+func TestAcc_View_columnsWithMaskingPolicyWithoutUsing(t *testing.T) {
+	t.Setenv(string(testenvs.ConfigureClientOnce), "")
+	_ = testenvs.GetOrSkipTest(t, testenvs.EnableAcceptance)
+	acc.TestAccPreCheck(t)
+
+	id := acc.TestClient().Ids.RandomSchemaObjectIdentifier()
+	table, tableCleanup := acc.TestClient().Table.CreateWithColumns(t, []sdk.TableColumnRequest{
+		*sdk.NewTableColumnRequest("id", sdk.DataTypeNumber),
+		*sdk.NewTableColumnRequest("foo", sdk.DataTypeNumber),
+		*sdk.NewTableColumnRequest("bar", sdk.DataTypeNumber),
+	})
+	t.Cleanup(tableCleanup)
+	statement := fmt.Sprintf("SELECT id, foo FROM %s", table.ID().FullyQualifiedName())
+
+	maskingPolicy, maskingPolicyCleanup := acc.TestClient().MaskingPolicy.CreateMaskingPolicyWithOptions(t,
+		[]sdk.TableColumnSignature{
+			{
+				Name: "One",
+				Type: sdk.DataTypeNumber,
+			},
+		},
+		sdk.DataTypeNumber,
+		`
+case
+	when One > 0 then One
+	else 0
+end;;
+`,
+		new(sdk.CreateMaskingPolicyOptions),
+	)
+	t.Cleanup(maskingPolicyCleanup)
+
+	projectionPolicy, projectionPolicyCleanup := acc.TestClient().ProjectionPolicy.CreateProjectionPolicy(t)
+	t.Cleanup(projectionPolicyCleanup)
+
+	// generators currently don't handle lists of objects, so use the old way
+	basicView := func(columns ...string) config.Variables {
+		return config.Variables{
+			"name":      config.StringVariable(id.Name()),
+			"database":  config.StringVariable(id.DatabaseName()),
+			"schema":    config.StringVariable(id.SchemaName()),
+			"statement": config.StringVariable(statement),
+			"column": config.SetVariable(
+				collections.Map(columns, func(columnName string) config.Variable {
+					return config.MapVariable(map[string]config.Variable{
+						"column_name": config.StringVariable(columnName),
+					})
+				})...,
+			),
+		}
+	}
+
+	basicViewWithPolicies := func() config.Variables {
```

Comment:
Is there a point in having two functions that return config when only one is used? If not, please merge them into one.

Reply:
Done.